.github/workflows/publish.yml

```
@@ -29,13 +29,41 @@ jobs:
         run: ./gradlew qualifyVersionGha
       - name: run checks
         id: checks
-        run: ./gradlew check
+        run: ./gradlew check -Pjunit-tags=!slow -x jcstress
```

```suggestion
        run: ./gradlew check -Pjunit-tags=!slow,!loop -x jcstress
```

found some places with loop tag but without slow

---

I think I'd better ensure the "slow" tag is everywhere and the "race" vs "loop" tags to further differentiate

---

than yeah. we need to mark race tests as slow as well

---

changed that (again) to have `Tag("slow")` identify ONLY the tests that we want in the parallel job (ie. raceXxx tests in CommonPoolTest)